doc/source/metadata/Introduction.rst

```diff
@@ -491,13 +483,13 @@ percentile
 Main percentile variable 
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-In the example here, the main variable is a 12 hour maxiumum temperature,
-but representing the nth percentile of the temperature forecast probability distribution
+In the example here, the main variable is a 12 hour maximum temperature,
+but representing the Nth percentile of the temperature forecast probability distribution
 at each point. 
 
 .. literalinclude:: temp12max_perc_ncdump.txt
     :tab-width: 4
-    :lines: 8-14
+    :lines: 8-14s
```

Comment:
I think this addition of a trailing `s` was a mistake.

Reply:
Indeed - probably CNTL S without the CNTL. Now corrected.